### PR TITLE
Update Cake to version 0.28.0

### PR DESCRIPTION
**Pull request by joaoasrosa**
This PR updates the Cake dependencies to latest version, avoiding to skip the validation if the user is using the latest version.

Part of #9

### CODE REVIEW

File: src/Cake.Yaml.Tests/Cake.Yaml.Tests.csproj [hunk 1]
```diff
@@ -5,8 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="0.26.0" />
-    <PackageReference Include="Cake.Testing" Version="0.26.0" />
+    <PackageReference Include="Cake.Core" Version="0.27.1" />
```
[the remark this answers is not in the document]
Comment (devlead) [reply src/Cake.Yaml.Tests/Cake.Yaml.Tests.csproj]: Latest breaking change was 0.26.0, so it should work with 0.27.1 already without needing to skip verification.

**Comment by pascalberger** [comment]
Updated the PR to build against latest Cake version containing breaking changes (0.28.0)

**Comment by pascalberger** [comment]
Failing build will be fixed once https://github.com/cake-contrib/Cake.Yaml/pull/11 is merged and this branch is rebased.

**Comment by pascalberger** [comment]
@joaoasrosa your changes have been merged, thanks for your contribution 👍

**Pull request merged**